crates/miden-objects/src/account/storage/slot/slot_name_id.rs

```diff
@@ -1,25 +1,49 @@
 use core::cmp::Ordering;
+use core::fmt::Display;
 
 use crate::Felt;
 
-// TODO(named_slots): Docs + separators for the entire module.
+/// The partial hash of a [`SlotName`](super::SlotName).
+///
+/// The ID of a slot name are the first (`suffix`) and second (`prefix`) field elements of the
+/// blake3-hashed slot name.
+///
+/// The slot name ID is used to uniquely identify a storage slot and is used to sort slots in
+/// account storage.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct SlotNameId {
-    prefix: Felt,
     suffix: Felt,
+    prefix: Felt,
 }
```

Comment:
I thought we were going to rename this to `StorageSlotId` - or is this for a follow-up?

Also, why the change in order of the fields?

Reply:
Yes, that's a follow-up.



> Also, why the change in order of the fields?

Felt more consistent to have the suffix, prefix order which matches how we have it laid out in memory, but no other strong reason.